scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt

```diff
@@ -61,7 +62,7 @@ import org.ossreviewtoolkit.utils.test.readResource
 
 class ScannerIntegrationFunTest : WordSpec({
     "Scanning all packages corresponding to a single VCS" should {
-        val analyzerResult = createAnalyzerResult(pkg0, pkg1, pkg2, pkg3, pkg4)
+        val analyzerResult = createAnalyzerResult(project0, pkg0, pkg1, pkg2, pkg3, pkg4)
```

Comment:
Would we get an easier to understand diff, if we extract adding the project to a separate (first) commit?